CDCgov/prime-reportstream
# Victor/8320 AOE questions FHIR to HL7 translation

This PR adds FHIR to HL7 translation support for AOE questions. AOE questions will come as an Observation under supportingInfo in a ServiceRequest and will get mapped to an OBX segment in HL7 v2. AOE questions will be mapped at the end of all other observations.

Test Steps:
1. Build this branch and run the unit and integration tests. Verify they all pass. Alternatively you can verify the Github Actions build passes. 
2. A new sample FHIR file (sample_AOE_1_20230220-0001.fhir) was added that contains  two AOE questions. 
3. Run:  `./prime fhirdata --input-file src/testIntegration/resources/datatests/FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir -s metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml --output-format HL7 --output-file src/testIntegration/resources/datatests/FHIR_to_HL7/sample_AOE_1_20230220-0001.hl7 ` 
4. Verify AOE questions are being mapped correctly at the end of all OBX segments for the specific OBR segment.

## Changes
- Updated FHIR to HL7 v2 mappings
- Added new system mapping for http://terminology.hl7.org/ValueSet/v2-0136

## Checklist

### Testing

## Linked Issues
- Fixes #8320
